bin/strap.sh

```diff
@@ -279,8 +287,8 @@ fi
 logk
 
 # Install Homebrew Bundle, Cask and Services tap.
-log "Installing Homebrew taps and extensions:"
-brew bundle --file=- <<EOF
+logn "Installing Homebrew taps and extensions..."
+brew bundle --file=- 1>/dev/null 2>&1 <<EOF
```

Comment:
we do want to be able to see this when we're in debug mode though....

Reply:
i think.... maybe not though

Reply:
Update.